src/main/java/eu/dissco/backend/domain/FdoType.java

```diff
@@ -0,0 +1,39 @@
+package eu.dissco.backend.domain;
+
+import com.fasterxml.jackson.annotation.JsonAlias;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+
+@Getter
+public enum FdoType {
```

Comment:
Ok this list is final right? This is what we will use when we move to production

Reply:
those 2 things aren't the same 😆 It's what we'll use for production though